packages/protocol/contracts/governance/Governance.sol

```diff
@@ -615,30 +650,63 @@ contract Governance is
    * @notice Approves the hash of a hotfix transaction(s).
    * @param hash The abi encoded keccak256 hash of the hotfix transaction(s) to be approved.
    */
-  function approveHotfix(bytes32 hash) external hotfixNotExecuted(hash) onlyApprover {
-    hotfixes[hash].approved = true;
-    emit HotfixApproved(hash);
+  function approveHotfix(bytes32 hash) external hotfixNotExecuted(hash) {
+    require(msg.sender != address(0), "msg.sender cannot be address zero");
+    require(
+      msg.sender == approver || msg.sender == securityCouncil,
+      "msg.sender not approver or Security Council"
+    );
+
+    if (msg.sender == approver) {
+      hotfixes[hash].approved = true;
+    } else {
+      if (isL2()) {
+        hotfixes[hash].councilApproved = true;
+      } else {
+        revert("Hotfix approval by security council is not available on L1.");
+      }
+    }
+    emit HotfixApproved(hash, msg.sender);
```

Comment:
would we want ta different event for security council approval vs regular approval?

Reply:
We could have a different one, but I felt that having the different address included in the event would take care of differenciating the two.